README/ReleaseNotes/v610/index.md

```diff
@@ -248,6 +248,7 @@ The following interfaces have been removed, after deprecation in v6.08.
 ## Parallelism and PROOF
 - Add ROOT::GetImplicitMTPoolSize function to get the size of the pool used to enable implicit multi threading
 - Add the TThreadExecutor::Foreach method for parallelising functions featuring void return type
+- Add TBufferMerger and TBufferMergerFile to facilitate writing in parallel to a single output file
```

Comment:
This is an 'important' new feature.  Should we consider expanding this text to include a short example (i.e. yes, the user could go and search for the reference guide and tutorials but I think it deserves to have a direct explanation here). thanks.